ci/cscs-ci.yml

```diff
@@ -1,6 +1,9 @@
 include:
   - remote: 'https://gitlab.com/cscs-ci/recipes/-/raw/master/templates/v2/.ci-ext.yml'
 
+.py312: &py312
```

Comment:
Why not changing 3.11 to 3.12 instead? Adding more jobs to CI will make the CI queue grow even longer...